### PR TITLE
ios updates

### PR DESCRIPTION
**Pull request by davidbielik**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 1]
```diff
@@ -6,27 +6,34 @@ platform: iOS
...
+### Upgrade Summary
 
-We expect to release our official support for iOS 14 soon after Apple releases their final version of iOS 14 (known as the "Golden Master").
+There are two necessary upgrades that are required to continue using Braze messaging features on iOS.
 
-If you experience any issues or questions related to our iOS 14 compatibility or beta release, please open a new [Github Issue][2].
+#### Location Targeting
 
-### Upgrade Summary
+Customers who are using the _Most Recent Location_ targeting features must upgrade to iOS SDK v3.26.1 which contains updates related to the new _Approximate Location_ feature in iOS 14. Even though your app might not target iOS 14, your end users may upgrade their devices which enable this new location accuracy option. Apps that do not upgrade to iOS SDK v3.26.1+ will not be able to collect location attributes when the _Approximate Location_ is selected by users on iOS 14 devices.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:26]:
```suggestion
Customers who are using the _Most Recent Location_ targeting features must upgrade to at least iOS SDK v3.26.1 which contains updates related to the new _Approximate Location_ feature in iOS 14. Even though your app might not target iOS 14, your end users may upgrade their devices which enable this new location accuracy option. Apps that do not upgrade to iOS SDK v3.26.1+ will not be able to collect location attributes when the _Approximate Location_ is selected by users on iOS 14 devices.
```

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 2]
```diff
@@ -6,27 +6,34 @@ platform: iOS
...
+
+#### iOS 14 + Xcode 12
+
+We have released [Beta versions][1] of our upcoming iOS 14 / Xcode 12 compatible SDK. Customers who relase Beta versions of their iOS apps with iOS 14 support are encouraged to use Braze's iOS 14 Beta SDK version which includes Xcode 12 compatibility and support for new iOS 14 APIs. We will continue to release updates and fixes to future beta releases as Apple continues to release newer versions of iOS 14 beta. If you experience any issues or questions related to our iOS 14 compatibility or beta release, please open a new [Github Issue][2].
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:30]:
```suggestion
We have released [Beta versions][1] of our upcoming iOS 14 / Xcode 12 compatible SDK. Customers who relase Beta versions of their iOS apps with iOS 14 support are encouraged to use Braze's iOS 14 Beta SDK version which includes Xcode 12 compatibility and support for new iOS 14 APIs. 

We will continue to release updates and fixes to future beta releases as Apple continues to release newer versions of iOS 14 beta. If you experience any issues or questions related to our iOS 14 compatibility or beta release, please open a new [Github Issue][2].
```

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 3]
```diff
@@ -6,27 +6,34 @@ platform: iOS
...
+
+Our official iOS 14 compatible SDK will be released shortly after Apple's final iOS 14 beta release, known as the "Golden Master" release.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:32]:
```suggestion
Our official iOS 14 compatible SDK will be released shortly after Apple's final iOS 14 beta is released, known as the "Golden Master" release.
```

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 4]
```diff
@@ -6,27 +6,34 @@ platform: iOS
...
 
-For apps targeting iOS 14, the following features have a change in behavior, or will require upgrading to the latest Braze iOS SDK.
+#### IDFA permissions
 
-- Geofences are not supported for users who choose the new  _approximate location_ permission.
-- Using the "Last Known Location" targeting functionality will require upgrading your Braze SDK to work with the new _approximate location_ permission.
-- Expect a sudden decrease in IDFA availability which now requires a permission prompt.
+Once users upgrade to iOS 14, apps that collect the IDFA advertising ID must use the new iOS 14 APIs to prompt users for IDFA permission. Failure to update your code will result in a blank IDFA value (`00000000-0000-0000-0000-000000000000`), similar to when a user declines this new prompt.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:36]:
```suggestion
Once users upgrade to iOS 14, apps that collect the IDFA advertising ID must use the new iOS 14 APIs to prompt users for IDFA permission. Failure to update your code will result in a blank IDFA value (`00000000-0000-0000-0000-000000000000`), similar to a user declining this new prompt.
```

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 5]
```diff
@@ -6,27 +6,38 @@ platform: iOS
...
 
-For a complete list of the new features and changes announced this year at WWDC, see Apple's [iOS 14 Preview](https://www.apple.com/ios/ios-14-preview/).
+For a complete list of new iOS 14 updates announced this year at WWDC, see Apple's [iOS 14 Preview](https://www.apple.com/ios/ios-14-preview/).
 
-## Braze SDK Compatibility
+#### Summary of iOS 14 breaking changes
 
-Apps that are targeting iOS 14 are required to upgrade Braze's iOS SDK to continue using messaging features. 
+- Geofences are no longer supported by iOS for users who choose the new  _approximate location_ permission.
+- Customers using the "Last Known Location" targeting features are required to upgrade their Braze iOS SDK to at least v3.26.1 in order to accept the new _approximate location_ permission.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:16]: in order to accept the new -> for compatibility with

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 6]
```diff
@@ -6,27 +6,38 @@ platform: iOS
...
 # iOS 14 SDK Upgrade Guide
 
-This guide describes relevant changes introduced in iOS 14 and the required upgrade steps for your Braze iOS SDK integration.
+This guide describes Braze-related changes introduced in iOS 14 and the required upgrade steps for your Braze iOS SDK integration.
 
-For a complete list of the new features and changes announced this year at WWDC, see Apple's [iOS 14 Preview](https://www.apple.com/ios/ios-14-preview/).
+For a complete list of new iOS 14 updates announced this year at WWDC, see Apple's [iOS 14 Preview](https://www.apple.com/ios/ios-14-preview/).
 
-## Braze SDK Compatibility
+#### Summary of iOS 14 breaking changes
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:13]: This is a mix of Braze related changes and iOS related changes, may be good to break apart

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 7]
```diff
@@ -6,27 +6,38 @@ platform: iOS
...
-We expect to release our official support for iOS 14 soon after Apple releases their final version of iOS 14 (known as the "Golden Master").
+There are two necessary upgrades that are required to continue using Braze messaging features on iOS.
 
-If you experience any issues or questions related to our iOS 14 compatibility or beta release, please open a new [Github Issue][2].
+#### Location Targeting
 
-### Upgrade Summary
+Customers who are using the _Most Recent Location_ targeting features must upgrade to at least iOS SDK v3.26.1 which contains updates related to the new _Approximate Location_ feature in iOS 14. 
+
+Even though your app might not target iOS 14, your end users may upgrade their devices which adds this new location accuracy option. Apps that do not upgrade to iOS SDK v3.26.1+ will not be able to collect location attributes when users provide their _approximate location_  on iOS 14 devices.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:28]: will not be able to collect -> will not be able to reliably collect

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md (hunk 6 above)
Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:13]: 🐃

**Pull request merged**